client/data/disputes/hooks.ts

```diff
@@ -30,6 +36,8 @@ export const useDisputeEvidence = () => {
 	return { updateDispute };
 };
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Why do we need those suppressions?

Reply:
Sorry, should have indicated WIP. I just put them so I could push and resolve later.

Reply:
Ok, I see. NP, we should drop them once the changes pass linters.